app/assets/javascripts/application.js

```diff
@@ -16,3 +16,11 @@
 //= require_tree .
 
 $(function(){ $(document).foundation(); });
+
+function toggle_visibility(id) {
```

Comment:
Identifier 'toggle_visibility' is not in camel case.<br>'toggle_visibility' is defined but never used.